.changeset/tame-ligers-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/api-generator": major
+---
+
+Api generator is now in separated `@comet/api-generator` package and can be run with the same command as before `comet-api-generator` command.
```

Comment:
```suggestion
Move API Generator into separate `@comet/api-generator` package

It can be run with the same `comet-api-generator` command as before.
```